webapp/src/main/resources/logback/logback-json.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <include resource="org/springframework/boot/logging/logback/defaults.xml" />
```

Comment:
and here just include `logback.xml` and then have the appender?

Reply:
We can't do it that way, because by importing `org/springframework/boot/logging/logback/base.xml` we'll be importing `org/springframework/boot/logging/logback/defaults.xml`, then overriding the `RollingFileAppender` already defined in the defaults.xml logback file, and when we do that the application doesn't start.
So on one end we need `logback.xml` to be the default logging config, and on the other end the `logback-json.xml` file to be the entrypoint when json logging is required.